sigmastate/src/main/scala/sigmastate/interpreter/Interpreter.scala

```diff
@@ -193,9 +196,17 @@ trait Interpreter extends ScorexLogging {
                     context: CTX,
                     env: ScriptEnv): (SigmaBoolean, Long) = {
     implicit val vs: SigmaValidationSettings = context.validationSettings
-    val prop = propositionFromErgoTree(ergoTree, context)
-    val (propTree, context2) = trySoftForkable[(BoolValue, CTX)](whenSoftFork = (TrueLeaf, context)) {
-      applyDeserializeContext(context, prop)
+
```

Comment:
move this to another PR, this change is not explained in PR description, also, it is not a good idea in general to mix changes in consensus-critical code with "todo coverage"

Reply:
rolled it back